wrappers/rust/icicle-core/src/msm/tests.rs

```diff
@@ -178,6 +179,172 @@ where
         .unwrap();
 }
 
+pub fn check_msm_batch_shared<C: Curve + MSM<C>>()
+where
+    <C::ScalarField as FieldImpl>::Config: GenerateRandom<C::ScalarField>,
+{
+    // let test_sizes = [1000, 1 << 16]; //TODO - uncomment this line after implementing fast msm
```

Comment:
hopefully we merge CPU msm today then let's remove those TODOs and merge this one too